modules/webp-uploads/load.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * WebP-Uploads Module
+ *
+ * @package performance-lab
+ * @since 1.0.0
+ *
+ * Module Name: WebP uploads.
```

Comment:
Nitpick, but can we unify the title case here so that it's consistent? Maybe `WebP Uploads`?